garrysmod/gamemodes/terrortown/gamemode/lang/russian.lua

```diff
@@ -88,7 +88,7 @@ L.equip_spec_name  = "Имя"
 L.equip_spec_type  = "Тип"
 L.equip_spec_desc  = "Описание"
 
-L.equip_confirm    = "Купить оборудование"
+L.equip_confirm    = "Купить"
```

Comment:
If something doesn't fit in the UI, perhaps the UI should be adjusted so the localized text can fit.